src/Main.elm

```diff
@@ -3,77 +3,432 @@ module Main exposing (main)
 import Browser
 import Data
 import Html exposing (Html)
-import Html.Attributes as Attrs
+import Json.Decode
+import Element as E exposing (Element)
+import Element.Font as F
+import Ui.Theme as T
+import Ui
+import Validate
 
 
+{-
 
----- MODEL ----
+TODO
+
+- Debounce so errors show up after finished typing.
+- Arrange validations errors such that submit button doesn't jump.
+- Highlight invalid input fields and add Aria.invalid tags.
+- Maybe refocus on invalid field when attempting to submit instead of
+  disabling submit button.
+- If submit button is to remain disabled, reconsider styling. Needs contrast,
+  but looks like secondary button currently.
+- Right now the value of the Inherited User field determines whether the
+  contact details can be edited. Alternatively, one could add a checkbox
+  to trigger the Inherited User field and choosing to inherit would feel
+  more intentional.
+- Add autofill attributes to e-mail and phone
+- This form doesn't actually submit anything, but it did, it should have a
+  success message of some sort.
+- Prettier responsibility.
```

Comment:
What does this mean?

Reply:
Oh, meant to say responsiveness! Thought it sounded weird at the time, but my late night mind couldn't figure out what was wrong haha